### ISSUE TITLE
[IBM Power Up] Your Project Title

### ISSUE THREAD

**Issue by Yiningxie** [XanaduAI/QHack2022#52]
### Team Name: QH

*Your team's name (matching the name used on the QHack Coding Challenges, if applicable)*

### Project Description: Integration of Quantum Computers with Quantum Communication

*A brief description of your project (1-2 paragraphs).*

As in the public news, the first Quantum Satellite was successfully launched in 2016, and the first Quantum Communication was carried out in 2017 between Austria and China. Recently the entanglement between 2 places with a distance over 1000 kilometers was detected. This project is to attempt  integration Quantum Computers with Quantum Communication.

My high school teacher was actually highly encouraging and recommending me to study theoretic physics with my excellent school performance and strong interest in physics,.  After my honorable graduation from Tongji Medical School at Huazhong University of Science and Technology and my Residency on Internal Medicine at Union (Xie He) Hospital.  I went to Germany and did my doctoral thesis at Munich University in the interest of science. After obtaining my doctoral degree with honor from Munich University, I came to University of California at Los Angeles (UCLA) for my post-doctoral fellowship and subsequently worked there. I also took and successfully passed the U.S. National Board Step-1, National Board Step-2 and Clinical Skill Assessment (CSA), and am certified by the U.S. Educational Commission for Foreign Medical Graduates (ECFMG).

Part of my post-doctoral fellowship at UCLA was working with an engineer who designed an electronic device controlled automatically by computers to work with biologic systems, for which I went back to a community college in Los Angeles on my own to learn Fortran. In 2006, I moved from Los Angeles (UCLA) to Washington, DC for my job at the National Institutes at Health (NIH). I have strengthened my interests in physic and also expanded my areas in technologies, including quantum communication and quantum computers. My e-mail is [yiningxie3@gmail.com](mailto:yiningxie3@gmail.com) and my phone # is 240-453-1534.  I am looking forward to hearing from you about my draft of this QHack Open Hackathon project.

Many thanks, Sincerely,

Yining Xie

### Source code: 

The Quantum Programming Language Twist from MIT will be used in conjunction with PenneLane (Pythone, Numby, PyTorch, TensorFlow, JAX interface, Cirq, Strawberrybields, Qiskitt, Forest, etc).

*A hyperlink to the draft source code for your team's hackathon project (e.g., a GitHub repo).*

https://doi.org/10.1145/3498691
Two Attachments here: 3498691.pdf; appendices.pdf
[3498691.pdf](https://github.com/XanaduAI/QHack/files/8121961/3498691.pdf)
[appendices.pdf](https://github.com/XanaduAI/QHack/files/8121962/appendices.pdf)

### Resource Estimate: 

If awarded, the access to IBM Quantum machine with IBM 16-qubit QPU will be used to finish this project.

The Quantum Programming Language Twist from MIT will be used in conjunction with PenneLane (Pythone, Numby, PyTorch, TensorFlow, JAX interface, Cirq, Strawberrybields, Qiskitt, Forest, etc) as stated the above.

*A 1-2 paragraph written Resource Estimate, indicating how you expect to use the IBM 16-qubit QPU, if awarded, to finish your Open Hackathon project.*

**Comment by isaacdevlugt** [comment]
Thank you for your Power Up submission! As a reminder, the final deadline for your project is February 25 at 17h00 EST. Submissions should be done here: https://github.com/XanaduAI/QHack/issues/new?assignees=&labels=&template=open_hackathon.md&title=%5BENTRY%5D+Your+Project+Title

This issue will be closed shortly.

Good luck!
